src/main/scala/dpla/ingestion3/utils/Utils.scala

```diff
@@ -63,7 +50,7 @@ object Utils {
     * @param runtime Runtime in milliseconds
     * @param recordCount Number of records output
     */
-  def logResults(runtime: Long, recordCount: Long, logger: Logger): Unit = {
+  def logResults(runtime: Long, recordCount: Long): String = {
```

Comment:
Could we change the name of this method since it isn't actually logging anything anymore?

Reply:
good call

Reply:
Actually, I think I renamed this in the other massive PR...

https://github.com/dpla/ingestion3/blob/1918cf27001a1cc1a32e73db54bbff273a50710e/src/main/scala/dpla/ingestion3/utils/HttpUtils.scala

Reply:
:+1: great minds... ;)